packages/now-node/src/helpers.ts

```diff
@@ -72,6 +73,46 @@ function getCookieParser(req: NowRequest) {
   };
 }
 
+function parseQSArray(key: string, value: any, acc: any): void {
```

Comment:
```suggestion
function parseQSArray(key: string, value: string, acc: any): void {
```

I think value is a string according to your calling function